content-src/lib/selectLayoutRender.js

```diff
@@ -38,6 +38,34 @@ export const selectLayoutRender = (state, rickRollCache) => {
   }
 
   const positions = {};
+  const filterComponentsByPref = () => {
+    const DS_COMPONENTS = ["Message", "SectionTitle", "Navigation",
+      "CardGrid", "Hero", "HorizontalRule", "List"];
+
+    const filterArray = [];
+
+    if (!prefs["feeds.topsites"]) {
+      filterArray.push("TopSites");
+    }
+
+    if (!prefs["feeds.section.topstories"]) {
```

Comment:
Mochitest is probably failing because this pref is false causing nothing to be rendered.